src/Car.java

```diff
@@ -0,0 +1,61 @@
+public class Car {
+    private String make = "Tesla";
+    private String model = "Model X";
+    private String color = "Gray";
+    private int doors = 2;
+    private boolean convertible = false;
+
+    public String getMake() {
+        return make;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getDoors() {
+        return doors;
+    }
+
+    public boolean isConvertible() {
+        return convertible;
+    }
+
+    public void setMake(String make) {
+       if (make == null) make = "Unknown";
+       String lowercaseMake = make.toLowerCase();
+        switch (lowercaseMake) {
+            case "holden", "porsche", "tesla" -> this.make = make;
```

Comment:
You used braces in `default` code block but here you didn't. Please be consist.

Reply:
Done

Reply:
When I used braces my cod dosen't work propely. I shouldn't do this

Reply:
This is a feature introduced in Java 12 known as "Switch Expressions". What is your Java version configured in Intelij? File > Project Structure > SDK. You should work on Java 17(LTS version - Long term support) or even 21(the latest version)

Reply:
java 17.0.6 is my version of Java